.gitmodules

```
@@ -44,3 +44,7 @@
 [submodule "src/tools/rust-analyzer"]
 	path = src/tools/rust-analyzer
 	url = https://github.com/rust-analyzer/rust-analyzer.git
+[submodule "src/backtrace"]
+	path = src/backtrace
+	url = https://github.com/rust-lang/backtrace-rs
```

```suggestion
	url = https://github.com/rust-lang/backtrace-rs.git
```
See the urls above.
IIRC, the `.git`-less urls have issues in some circumstances so people always send follow ups adding it after new submodules are added.

---

For reference: https://github.com/rust-lang/rust/pull/53959